source/user-guide/describing-information/deleting-metadata.rst

```diff
@@ -13,8 +13,5 @@ There are different ways to remove a record:
 - In the ``contribute`` section click the ``delete`` button for a record.
```

Comment:
looks like highlighting of elements is not consistent. Sometimes double quotes are used, and others you use asterisks. Maybe better to use :guilabel:`[element]` to have a more consistent approach and better semantics in the text